absearch/tests/test_metrics.py

```diff
@@ -58,3 +57,33 @@ def test_metrics():
 
     # we incremented on redis the counter for the default cohort
     assert len(stats.timers['redis.incr']) == 1
+
+
+def test_enrolled():
+    app = get_app()
+    stats = app.app._statsd
+
+    # get a cohort
+    cohort = 'default'
+    while cohort == 'default':
+        path = '/1/firefox/39/beta/cs-CZ/cz/default/default'
+        res = app.get(path)
+        cohort = res.json.get('cohort', 'default')
+
+    # we got one cohort counter
+    assert stats.counters['enrolled'] == 1
+
+    # we called add_user_to_cohort once
+    assert len(stats.timers['add_user_to_cohort']), 1
+
+    # now that we have a cohort let's check back the settings
+    path = '/1/firefox/39/beta/cs-CZ/cz/default/default/' + cohort
+    res = app.get(path)
+
+    assert stats.counters['refresh'] == 1
+
+    # also, an unexistant cohort should fall back to the default
+    # and count as a discard as well
+    path = '/1/firefox/39/beta/cs-CZ/cz/default/default/meh'
+    app.get(path)
+    assert stats.counters['discarded'] == 1
```

Comment:
it's asserted here : https://github.com/mozilla-services/absearch/blob/master/absearch/tests/test_server.py#L107

Maybe I should reorganize the tests a little bit (lots of dupes)